### PR TITLE
Restyled the components

### PR DESCRIPTION
**Pull request by demonicirfan**
Solves issue #20

Completed Tasks
- Website looks the same as the design and is responsive.
- Responsive carousel.
- Used Bootstrap for most of the part.

Remaining Tasks
- I am working on Navbar, I am trying to make it look as similar to the design as I can, it will take some time.
- I was not able to use custom colors in bootstrap so I used CSS for colors. I will figure it out and use custom colors it will reduce the amount of CSS.
- Consistent text styles and sizes on the webpage are still to be done. Will do that with colors.
- Used dummy data in the community activity section.

### CODE REVIEW

**Comment by Sing-Li** [comment]
> I am working on Navbar, I am trying to make it look as similar to the design as I can, it will take some time.

Awesome.  Thanks for submitting it WIP early.  This will really help with the review.    

Please be aware that we will be adding two sub-components to it very shortly:
1) rightmost will be a square auth component  -- interchangeably it can be supported by Firebase/auth0/gluu/keycloak component(s)
2) second right most will be a "3 dots drop down"  Superprofile component

**Comment by demonicirfan** [comment]
Ok, so we can style it with CSS because bootstrap does not provide much flexibility for customization. Later on we can add those two sub-components.

**Comment by Sing-Li** [comment]
@demonicirfan  I ran a test on the checkout of the PR and other than the carousel, the styling of the other components don't seem to be working.   For example, the count-ups don't reactively reposition correctly.  The persona tiles are still white text on blue and does not show red circle with icon.  The search bar still has no dropdown and spans the entire width.

Please let me know when the next stable  checkpoint is reached on the PR.

File: app/components/discourserankedlist.js [hunk 1]
```diff
@@ -1,55 +1,187 @@
...
...h3.5c.937 0 1.599-.477 1.934-1.064a1.86 1.86 0 0 0 .254-.912c0-.152-.023-.312-.077-.464.201-.263.38-.578.488-.901.11-.33.172-.762.004-1.149.069-.13.12-.269.159-.403.077-.27.113-.568.113-.857 0-.288-.036-.585-.113-.856a2.144 2.144 0 0 0-.138-.362 1.9 1.9 0 0 0 .234-1.734c-.206-.592-.682-1.1-1.2-1.272-.847-.282-1.803-.276-2.516-.211a9.84 9.84 0 0 0-.443.05 9.365 9.365 0 0 0-.062-4.509A1.38 1.38 0 0 0 9.125.111L8.864.046zM11.5 14.721H8c-.51 0-.863-.069-1.14-.164-.281-.097-.506-.228-.776-.393l-.04-.024c-.555-.339-1.198-.731-2.49-.868-.333-.036-.554-.29-.554-.55V8.72c0-.254.226-.543.62-.65 1.095-.3 1.977-.996 2.614-1.708.635-.71 1.064-1.475 1.238-1.978.243-.7.407-1.768.482-2.85.025-.362.36-.594.667-.518l.262.066c.16.04.258.143.288.255a8.34 8.34 0 0 1-.145 4.725.5.5 0 0 0 .595.644l.003-.001.014-.003.058-.014a8.908 8.908 0 0 1 1.036-.157c.663-.06 1.457-.054 2.11.164.175.058.45.3.57.65.107.308.087.67-.266 1.022l-.353.353.353.354c.043.043.105.141.154.315.048.167.075.37.075.581 0 .212-.027.414-.075.582-.05.174-.111.272-.154.315l-.353.353.353.354c.047.047.109.177.005.488a2.224 2.224 0 0 1-.505.805l-.353.353.353.354c.006.005.041.05.041.17a.866.866 0 0 1-.121.416c-.165.288-.503.56-1.066.56z' />
```
Comment (Sing-Li) [inline app/components/discourserankedlist.js:158]: Please move all the in-line SVG to `app/public`.    We'll need a strategy to keep them with the associated component in the longer run.

File: app/components/personalcircle.js [hunk 2]
```diff
@@ -1,18 +1,145 @@
...
-              <Button>{persona.name}</Button>
-            </span>
-          ))}
+  return (
+    <>
+      <Col
+        className={`${styles.personas} d-flex flex-wrap justify-content-center`}
+      >
+        <span className={`${styles.persona}`}>
+          <div className={styles.svg}>
```
Comment (Sing-Li) [inline app/components/personalcircle.js:11]: pls move in-line svg as per earlier comment.

**Comment by demonicirfan** [comment]
> @demonicirfan I ran a test on the checkout of the PR. Everything looks good.
> 
> image
> 
> The one change I'd request is for the search drop-down to be on the left of the searchbar (even when reactive). And the Magnifying glass be on the right of the search entry.
> 
> Thanks! Please let me know when the next stable checkpoint is reached on the PR.

i have redesigned it you can have a look at the Figma file now and suggest if we should go with that. I have sent updates in the navbar-considerations chat

File: app/components/personalcircle.js (hunk 2 above)
Comment (demonicirfan) [reply app/components/personalcircle.js:11]: In the previous code, we were getting all the data used here ( like name and icons ) from backend, so we can replace the svg's there with these svg's directly and change the frontend code when that is done

**Comment by demonicirfan** [comment]
I will add all the svg's to `/public` folder while working on navbar. For now i have added these two svg's

**Review by Sing-Li — APPROVED** [review]
Good enough for initial commit.

**Pull request merged**